v2/parser/annotations/jsontags.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This is based on the highly efficient approach from https://golang.org/src/encoding/json/encode.go
+
+package annotations
```

Comment:
Why this package name?  It makes me think of kube annotations.  Why not `tags` or `fieldtags` or something?  Do you have bigger aspirations?

Reply:
Freudian slip. Java calls this sort of things annotations.    I'll use `tags`.